cha/introduction/sota_app.tex

```diff
@@ -0,0 +1,254 @@
+    \end{minipage}
+\end{figure}
+
+\iffalse
+By Amp Me Inc.
+4.2 stars on Google Play, current version, 3.5 stars, all versions 4 stars on App Store
+Functionality / How good it works
+30. januar 2017 (Android), Feb 01, 2017 (iOS)
+Android and iOS
+Spotify, SoundCloud, YouTube and your local music library
+Require internet, WiFi/Mobile data
+Free
+Resync button, syncs upon connection
+\fi
+
+\subsection{Chorus}\label{subsec:chorus}
+The final app is called Chorus\footnote{https://play.google.com/store/apps/details?id=com.avrapps.chorus} and is made by AVR APPS.
+The used version of the app is 2.1.
+It supports Android 4.0 or newer, AVR APPS mention an iOS app, but the App Store page returns that the app is not available in our region,
+when open in iTunes\footnote{https://itunes.apple.com/app/chorus/id894014439}.
+
+To share music between devices, WiFi or mobile hotspot is required,
+which means that all devices have to be on the same network.
+Chorus only supports playback of local media, so the music have to be in memory on the master device.
+
+When the app is opened, it looks like a regular music player, as seen on \cref{fig:chorus_screenshot}.
+Music is added by pressing the small plus sign in the right bottom corner. 
+When that music is added, Chorus streams the music to connected slave devices.
+To connect to a master device playing music, you press the menu button in the right top corner, and press join.
+Here a menu pops up with devices playing on the network, and if you select the device shown, it connects as a slave and starts playing.
+We encountered a scenario where the app just kept loading when attempting to join an active network, after a restart of the app it worked.
+
+It is intuitive to use the app, it is very minimalistic hence easy to navigate.
+The design is a bit old but its simplicity makes up for it.
+
+Chorus supports manual and automatic synchronization.
+On \cref{fig:chorus_screenshot} the synchronization slider can be seen,
+it slides from $-400ms$ to $+400ms$, and each step is $\Delta 10ms$.
+
+The app is free to download and use without any adds.\cite{chrous_play} 
+
+\begin{figure}[h!]
+    \setlength{\fboxsep}{0pt}
+    \setlength{\fboxrule}{2pt}
+    \begin{minipage}[b]{0.45\textwidth}
+        \centering
+        \fbox{\includegraphics[width=0.65\textwidth]{img/sota/chorus.png}}
+        \caption{A screenshot of Chorus when the app is opened.}\label{fig:chorus_screenshot}
+    \end{minipage}
+    \hfill
+    \begin{minipage}[b]{0.45\textwidth}
+        \centering
+        \fbox{\includegraphics[width=0.65\textwidth]{img/sota/chorus_slider.png}}
+        \caption{A screenshot of Chorus' synchronization slider.}\label{fig:chorus_slider}
+    \end{minipage}
+\end{figure}
+
+\iffalse
+By AVR APPS 
+3.3 stars on Google Play
+Functionality / How good it works
+11. maj 2015
+Android 4.0, iOS
+Local media?
+Mobile hotspot or WiFi
+Free
+Manual/automatic sync
+\fi
+\subsection{Determining the State of the Art}

```

Comment:
Savner generelt en vinkel som er brugbar for os. Der er fokus på user experience hvilket vi ikke rigtig kan bruge til noget i vores project, dette gælder også de enkelte paragrapher der er rundt omkring omkring minimalistisk og simplistisk design. - Conclusionen virker for mig som et app-review du finder på buzzfeed for at finde ud af hvilken app du skal vælge, from for at udmunde i noget som vi kan gå videre med i vores project